src/test/java/com/wikia/webdriver/testcases/discussions/Upvoting.java

```diff
@@ -49,8 +48,7 @@ public void anonUserOnMobileCanNotUpvote() {
    * ANONS ON DESKTOP SECTION
    */
 
-  @Test(groups = "discussions-anonUserOnDesktopCanNotVoteForPostDetails", enabled = false)
-  @RelatedIssue(issueID = "SOC-2596")
+  @Test(groups = "discussions-anonUserOnDesktopCanNotVoteForPostDetails", enabled = true)
   @Execute(asUser = User.ANONYMOUS)
   @InBrowser(browser = Browser.FIREFOX, browserSize = DESKTOP_RESOLUTION)
```

Comment:
browserSize does not work on Chrome, red more here :)
https://wikia-inc.atlassian.net/wiki/pages/viewpage.action?pageId=71303284